src/core/constants.js

```diff
@@ -169,6 +169,7 @@ export const SET_CLIENT_APP = 'SET_CLIENT_APP';
 export const SET_ERROR = 'SET_ERROR';
 export const SET_ERROR_MESSAGE = 'SET_ERROR_MESSAGE';
 export const SET_LANG = 'SET_LANG';
+export const SET_REQUEST_ID = 'SET_REQUEST_ID';
```

Comment:
I wish we had a duck for those constants and actions above.

Reply:
Yeah. I didn't want to rock the boat too much.